Model/QueuedTask.php

```diff
@@ -72,7 +72,7 @@ public function createSingletonJob($jobName, $data, $notBefore = null, $group =
 				'completed' => null,
 				'or' => array(
 					'fetched' => null,
-					'failed' => 0,
+					'failed >' => 0,
```

Comment:
I think this should be failed < 5  otherwise if a task ever failed out it won't ever be recreated